tests/console/yast2_kdump_use_firmware_assisted_dump.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Select Firmware-Assisted Dump and accept kdump options.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam suse de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+use cfg_files_utils qw(validate_cfg_file);
+use scheduler qw(get_test_suite_data);
+use testapi qw(save_screenshot select_console);
+use utils qw(systemctl zypper_call);
+use YaST::Module;
+
+sub run {
+    my $kdump = $testapi::distri->get_kdump();
+
+    select_console('root-console');
+    zypper_call('in kdump');
+
+    YaST::Module::open(module => 'kdump', ui => 'ncurses');
+
+    $kdump->use_firmware_assisted_dump();
+    save_screenshot;
+    $kdump->accept();
```

Comment:
Example without Controller:
```
my $need_reboot = $kdump->accept();
$need_reboot->confirm();
```

Reply:
or better:
`kdump->accept()->confirm_reboot_needed()`;
check error in stack trace is clear which method failed. simulate with a die inside the popup method.